docs/Indexing_JSON.md

```diff
@@ -70,6 +70,39 @@ FT.SEARCH userIdx '@name:(John)'
    2) "{\"user\":{\"name\":\"John Smith\",\"tag\":\"foo,bar\",\"hp\":1000,\"dmg\":150}}"
 ```
 
+## Indexing JSON arrays with tags
```

Comment:
Can this kindly be added also to the corresponding [page](https://oss.redis.com/redisjson/indexing_JSON/) on RedisJSON?

Reply:
See RedisJSON/RedisJSON#629 - I opened and have been maintaining them in parallel :)